app/models/git/exercise.rb

```diff
@@ -176,12 +183,12 @@ def important_absolute_filepaths
 
     memoize
     def cli_filepaths
-      special_filepaths = [SPECIAL_FILEPATHS[:readme], SPECIAL_FILEPATHS[:help]]
+      special_filepaths = [SPECIAL_FILEPATHS[:config], SPECIAL_FILEPATHS[:readme], SPECIAL_FILEPATHS[:help]]
```

Comment:
An alternative would be to do:

```suggestion
      special_filepaths = SPECIAL_FILEPATHS
```

and then delete the hints file on the next lineunless `filepaths.include?(hints_filepath)`

Reply:
Maybe this?
```
special_filepaths = SPECIAL_FILEPATHS.values_at(:config, :readme, :help)
```